src/SupportMessage/SupportMessage.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, ReactNode } from 'react'
+import styled, { css } from 'styled-components'
+
+import { Box } from '../Box'
+import { Icon } from '../Icon'
+import { theme } from '../theme'
+
+type BorderProps =
+  | { hasBorder?: false; borderColor?: never }
+  | { hasBorder: true; borderColor: string }
+
+export type SupportMessageProps = {
+  className?: string
+  children: ReactNode
+  type: 'info' | 'alert' | 'warning'
+  title?: string
+  hasBackground?: boolean
+} & BorderProps
+
+export const SupportMessage: FC<SupportMessageProps> = ({
+  className,
+  children,
+  type = 'info',
+  hasBorder,
+  borderColor,
+  title,
+  hasBackground = true,
+}) => (
+  <Wrapper
+    className={className}
+    type={type}
+    hasBorder={hasBorder}
+    borderColor={borderColor}
+    hasBackground={hasBackground}
+  >
+    <IconWrapper>
+      <Icon
+        size={24}
+        render={type}
+        color={
+          type === 'warning'
+            ? 'error'
+            : type === 'alert'
+            ? 'agentWarning'
+            : 'secondary'
+        }
+      />
+    </IconWrapper>
+    <Box flex direction="column">
+      <Title>{title}</Title>
+      {children}
+    </Box>
+  </Wrapper>
+)
+
+interface IWrapper {
+  type: 'info' | 'alert' | 'warning'
+  hasBorder?: boolean
+  borderColor?: string
+  hasBackground?: boolean
+}
+
+const IconWrapper = styled(Box)`
+  align-self: center;
+`
+
+const Wrapper = styled.div<IWrapper>(
+  ({ type, hasBorder, borderColor, hasBackground }) => css`
+    align-items: center;
+    ${hasBackground &&
+    `background-color: ${
+      type === 'info'
+        ? theme.colors.background
+        : type === 'alert'
+        ? theme.colors.bgSecondary
+        : '#FBEAEA'
```

Comment:
we shouldn't use colours that are not in our palette. I think this has to bee some sort of lighter version of `error`